python/py-imagehash/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-imagehash
+python.rootname     ImageHash
+version             4.3.1
+
+categories-append   devel graphics
+platforms           {darwin any}
+supported_archs     noarch
+license             custom
+maintainers         nomaintainer
+
+description         Perceptual Image Hashing Module
+long_description    Image hashes tell whether two images look nearly \
+                    identical. This is different from cryptographic \
+                    hashing algorithms (like MD5, SHA-1) where tiny \
+                    changes in the image give completely different \
+                    hashes. In image fingerprinting, we actually want \
+                    our similar inputs to have similar output hashes \
+                    as well. The image hash algorithms (average, \
+                    perceptual, difference, wavelet) analyse the image \
+                    structure on luminance (without color \
+                    information). The color hash algorithm analyses \
+                    the color distribution and black & gray fractions \
+                    (without position information).
+
+homepage            https://github.com/JohannesBuchner/imagehash
+
+checksums           md5     6c1788e8e8c48a3c0fa26566cef1a93b \
+                    rmd160  46c6a282f7ebb9c3fb5256a318259699fdb9e817 \
+                    sha256  7038d1b7f9e0585beb3dd8c0a956f02b95a346c0b5f24a9e8cc03ebadaf0aa70 \
+                    size    296989
+
+python.versions     38 39 310 311
+
+if {$subport ne $name} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    depends_run-append \
+                    port:py${python.version}-numpy \
+                    port:py${python.version}-pillow \
```

Comment:
`Pillow` should be capitalized as that's the portname; otherwise it will fail on case-sensitive filesystems

Reply:
Thanks! Done.